### ISSUE TITLE
Make code samples copy-paste friendly in some way

### ISSUE THREAD

**Issue by audreyr** [feldroy/django-crash-course#170]
## Location within the Book

Many locations, such as:

* Chapter or Appendix: 1. The Ultimate Django Setup
* Section: Step 4: Installing PostgreSQL
* Subsection: Installing PostgreSQL on Mac

## Description 

Code doesn't copy-paste well from the PDF. For example, when I try to copy-paste the command in section **Installing PostgreSQL on Mac**, I get a badly-formatted command:

```
sudomkdir -p /etc/paths.d&&\echo /Applications/Postgres.app/Contents/Versions/latest/bin|\sudotee /etc/paths.d/postgresapp
```

The command should be copy-pastable as:

```
sudo mkdir -p /etc/paths.d && echo /Applications/Postgres.app/Contents/Versions/latest/bin | sudo tee /etc/paths.d/postgresapp
```

## Possible Solutions

* Find a way to make copy-paste from the PDF work. (I haven't personally investigated, but @pydanny says this isn't possible due to the limitations of the PDF format.)
* Put code samples into a repo such as this one, and provide the copy-paste link below each code sample.
* Any other ideas? @luzfcb thoughts?

**Comment by pydanny** [comment]
For Two Scoops of Django, we wrote a code extractor for the LaTeX of the book. We should have something similar for Django Crash Course. We will be able to use it in following books.

The end result is what you see in this folder: https://github.com/twoscoops/two-scoops-of-django-1.11/tree/master/code

**Comment by pydanny** [comment]
Working on this now, it will be in a `code` directory in this repo hopefully on Monday.

**Comment by alqinae** [comment]
I got a nicely formatted command for the example above. However, when I tried to copy-paste it, I got `zsh: command not found: sudo`

I thought that the problem is with my terminal, but then I figured out that when I remove the second `\`, the command went through successfully.

The final command was:

> sudo mkdir -p /etc/paths.d && \
> echo /Applications/Postgres.app/Contents/Versions/latest/bin | sudo tee /etc/paths.d/postgresapp

Here is a picture of the appearance of command which shows that it is well formatted:
<img width="538" alt="Screen Shot 2020-03-16 at 10 50 07 AM" src="https://user-images.githubusercontent.com/8688806/76734404-e9d26080-6773-11ea-9296-585b89e60bc7.png">

I think just removing the second `\` which is before `sudo tee` will make it working.

That's for this example only, I don't know about other examples because I just started reading the book.
